tests/components/controlBar-test.js

```diff
@@ -1112,19 +1112,41 @@ describe('ControlBar', function() {
     expect(slider).not.toBe(null);
   });
 
-  it('hides the volume on iOS', function() {
-    OO_setWindowNavigatorProperty('platform', 'iPhone');
-
-    baseMockProps.skinConfig.buttons.desktopContent = [{'name':'volume', 'location':'controlBar', 'whenDoesNotFit':'keep', 'minWidth':100 }];
+  describe('Hides the volume btn on iOs', function() {
+    beforeEach(function() {
+      OO_setWindowNavigatorProperty('userAgent', 'phone');
+      OO_setWindowNavigatorProperty('platform', 'iPhone');
+      baseMockProps.skinConfig.buttons.desktopContent = [{'name':'volume', 'location':'controlBar', 'whenDoesNotFit':'keep', 'minWidth':100 }];
+    });
+    afterEach(function() {
+      OO_setWindowNavigatorProperty('userAgent', 'desktop');
+      OO_setWindowNavigatorProperty('platform', '');
+    });
 
-    var wrapper = Enzyme.mount(
-      <ControlBar {...baseMockProps} controlBarVisible={true}
+    var getControlBar = function(){
+      return (<ControlBar
+        {...baseMockProps}
+        controlBarVisible={true}
         componentWidth={500}
-        playerState={CONSTANTS.STATE.PAUSE}
-      />
-    );
+        playerState={CONSTANTS.STATE.PLAYING}
```

Comment:
Can we perform the test for both the PLAYING and PAUSED states?

Reply:
\+